src/software/thunderscope/robot_communication.py

```diff
@@ -58,18 +58,18 @@ def __init__(
         self.send_estop_state_thread = threading.Thread(target=self.__send_estop_state)
         self.run_thread = threading.Thread(target=self.run)
 
-        try:
-            self.estop_reader = ThreadedEstopReader(
-                self.estop_path, self.estop_buadrate
-            )
-        except Exception:
-            raise Exception("Could not find estop, make sure its plugged in")
+        # try:
+        #     self.estop_reader = ThreadedEstopReader(
+        #         self.estop_path, self.estop_buadrate
+        #     )
+        # except Exception:
+        #     raise Exception("Could not find estop, make sure its plugged in")
 
     def __send_estop_state(self):
         while True:
-            self.full_system_proto_unix_io.send_proto(
-                EstopState, EstopState(is_playing=self.estop_reader.isEstopPlay())
-            )
+            # self.full_system_proto_unix_io.send_proto(
+            #     EstopState, EstopState(is_playing=self.estop_reader.isEstopPlay())
+            # )
```

Comment:
It looks like some of the code was accidentally commented out!?

Reply:
Sorry about that, I was working on another issue and accidentally committed that file.